### PR TITLE
Adding Zoho Sites Template development preset to docs.

### PR DESCRIPTION
**Pull request by ringods**
* Adding a new section `Thirdparty Presets or Middleware` to the `Learning Resources` page
* Lists my preset that helps with Zoho Sites template development.

### CODE REVIEW

**Comment by edmorley** [comment]
This seems like a good place to put links like this to me, unless you can think of somewhere better @eliperelman? :-)

File: docs/learning-resources.md [hunk 1]
```diff
@@ -26,3 +26,9 @@ community, file an issue or submit a pull request.
 
 * [Bootstrap a Zero Configuration React App with Neutrino](https://egghead.io/lessons/react-bootstrap-a-zero-configuration-react-app-with-neutrino) by [@markshust](https://twitter.com/markshust)
 * [Empower and Simplify JavaScript Development](https://www.youtube.com/watch?v=fz5jMdmKmRI) by [@eliperelman](https://twitter.com/eliperelman) at [@nebraskajs](https://nebraskajs.com)
+
+## Thirdparty Presets or Middleware
```
Comment (eliperelman) [inline docs/learning-resources.md]: Can you make "Thirdparty" two words, e.g., "Third Party"? Also change the word "or" to "and".

Comment (ringods) [reply docs/learning-resources.md]: @eliperelman Done!

**Comment by eliperelman** [comment]
Thank you!

**Pull request merged**